crates/subspace-block-relay/src/protocol/compact_block.rs

```diff
@@ -209,23 +206,27 @@ impl<DownloadUnitId, ProtocolUnitId, ProtocolUnit>
     ProtocolClient<DownloadUnitId, ProtocolUnitId, ProtocolUnit>
     for CompactBlockClient<DownloadUnitId, ProtocolUnitId, ProtocolUnit>
 where
-    DownloadUnitId: Encode + Decode + Send + Clone + std::fmt::Debug,
-    ProtocolUnitId: Encode + Decode + Send + Clone,
-    ProtocolUnit: Encode + Decode + Send + Clone,
+    DownloadUnitId: Send + Sync + Encode + Decode + Send + Clone + std::fmt::Debug,
+    ProtocolUnitId: Send + Sync + Encode + Decode + Send + Clone,
+    ProtocolUnit: Send + Sync + Encode + Decode + Send + Clone,
```

Comment:
`Send` is twice here and it is usually at the end of trait bounds (not sure why, just happens to be the last thing people add when code doesn't compile I guess).